src/mmw/apps/modeling/models.py

```diff
@@ -84,7 +81,7 @@ def __unicode__(self):
 
     @property
     def in_drb(self):
-        return self.area_of_interest.within(DRB)
+        return self.area_of_interest.within(settings.DRB_SIMPLE_PERIMETER)
```

Comment:
Went with this which is actually a superior approximation:

![image](https://user-images.githubusercontent.com/1430060/85214727-15e61a80-b33d-11ea-8617-1d733becf58c.png)